src/compatTest/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -897,7 +897,10 @@ trait OrganizeImportsTest(val x: String):
         '3.0.2'      || _
         '3.1.3'      || _
         '3.2.2'      || _
-        '3.3.1'      || _
+        '3.3.4'      || _
+        '3.4.3'      || _
+        '3.5.2'      || _
+        '3.6.2'      || _
```

Comment:
we can probably drop all the older versions to keep the test runtime manageable.